mmcls/models/backbones/resnet.py

```diff
@@ -653,6 +653,21 @@ def train(self, mode=True):
                     m.eval()
 
 
+@BACKBONES.register_module()
+class ResNetV1c(ResNet):
+    """ResNetV1c variant described in [1]_. Compared with default
+    ResNet(ResNetV1b), ResNetV1c replaces the 7x7 conv in the input stem with
+    three 3x3 convs. For more details please refer to `Bag of Tricks for Image
+    Classification with Convolutional Neural Networks.
+
+    <https://arxiv.org/abs/1812.01187>`_.
+    """
```

Comment:
Please re-format the docstring according to the docstring of `ResNetV1d`

Reply:
done.